groupie/src/main/java/com/genius/groupie/GroupAdapter.java

```diff
@@ -53,13 +50,12 @@ public int getSpanCount() {
     }
 
     @Override public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
-        Item contentItem = getItem(position);
-        contentItem.bind(holder, position, onItemClickListener);
```

Comment:
another commit sounds good with a test or two, I will add an issue to track this :)